integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/CredentialCreateTest.java

```diff
@@ -75,11 +80,11 @@ public void testCreateCredentialWithZeroRoles(TestContext testContext) {
         useRealUmsUser(testContext, AuthUserKeys.ZERO_RIGHTS);
         testContext
                 .given(CredentialTestDto.class)
-                .when(credentialTestClient.create(), RunningParameter.key("Unauthorized"))
-                .expect(ForbiddenException.class,
-                        RunningParameter.expectedMessage("You have no right to perform " +
-                                "environments/createCredential in account 460c0d8f-ae8e-4dce-9cd7-2351762eb9ac")
-                                .withKey("Unauthorized"))
+                .whenException(credentialTestClient.create(), ForbiddenException.class,
+                        RunningParameter
+                                .expectedMessage("You have no right to perform environments/createCredential in account 460c0d8f-ae8e-4dce-9cd7-2351762eb9ac")
+                                .withKey("Unauthorized")
```

Comment:
A general comment, is it still needed to pack these behind keys? since with this new statement we are checking if the exception happened or not right away. Can we omit these keys with this implementation?